lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -93,14 +93,22 @@ export class PublicClientApplication extends ClientApplication implements IPubli
      * @returns {Promise.<AuthenticationResult>} - a promise that is fulfilled when this function has completed, or rejected if an error was raised. Returns the {@link AuthResponse} object
      */
     async acquireTokenSilent(request: SilentRequest): Promise<AuthenticationResult> {
+        // check to make sure a valid silent token retrieval stratgey was passed in, if applicable
+        if (request.silentTokenRetrievalStrategy && !Object.values(SilentTokenRetrievalStrategy).includes(request.silentTokenRetrievalStrategy as SilentTokenRetrievalStrategy)) {
```

Comment:
I suggest we move this check and error into a helper method. Something like `validateSilentTokenRetrievalStrategy()` that checks and throws if the strategy is set but not supported.

Reply:
I don't think we need this check. The enum clearly documents which values are expected, and downstream code can make explicit checks for each of those values, and a value that isn't one of those can be ignored.